portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,45 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  /* margin-left: auto;
+  margin-right: auto; */
+  width: 100%;
 }
```

Comment:
Delete the commented out properties here and in other places in the file.

Reply:
done